test/integration/features/cmd/version.feature

```diff
@@ -0,0 +1,7 @@
+@minishift-only @command
```

Comment:
I suggest to make it part of the naming, instead of a subfolder. Folders can always be introduced at a later stage